README.md

```diff
@@ -1,6 +1,8 @@
-# FreeRTOS PKCS #11 Library
+# corePKCS11 Library 
 
-This repository contains a software based PKCS #11 library to enable rapid development and flexibility when developing libraries and platforms that rely on crypto operations.
+This repository contains a software based PKCS #11 library that follows the [Oasis PKCS11](http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html) industry standard specification.  It is designed to enable rapid development and flexibility in use cases that rely on crypto operations
```

Comment:
made some updates to the language, we also want it to follow the standard paragraph for all other spoke repos